src/server/status_server.rs

```diff
@@ -210,6 +213,163 @@ impl StatusServer {
         Box::new(ok(res))
     }
 
+    fn frames_post_processor() -> impl Fn(&mut pprof::Frames) {
+        let thread_rename = [
+            (Regex::new(r"^grpc-server-\d*$").unwrap(), "grpc-server"),
```

Comment:
seem we don't need to create the Regex every time?

Reply:
I prefer you add a test for this function.

Reply:
Regex related function has been extracted as a standalone function for convenience to test. Several tests have been added at the bottom of this file.